### PR TITLE
Checked the README with Grammarly

### PR DESCRIPTION
**Pull request by renatgalimov**

### CODE REVIEW

**Review by vpetersson — APPROVED** [review]

**Comment by vpetersson** [comment]
Superseded by https://github.com/Screenly/playground/pull/8

**Comment by renatgalimov** [comment]
@vpetersson, I re-checked the README with Grammarly.

File: javascript-injectors/README.md [hunk 1]
```diff
@@ -104,4 +103,4 @@ For more details see the `window.ScrollBy()` [documentation](https://developer.m
 <img src="../images/engadget-logo.png" alt="Engadget logo" width="200"/>
 
 * Download [engadget-cookies-consent-closing.js](https://github.com/Screenly/playground/tree/master/javascript-injectors/engadget-cookies-consent-closing.js)
-* PATCH the asset.
+* PATCH the .
```
Comment (vpetersson) [inline javascript-injectors/README.md:106]: Accidental delete?

**Review by vpetersson — CHANGES_REQUESTED** [review]
Minor change

File: javascript-injectors/README.md (hunk 1 above)
Comment (renatgalimov) [reply javascript-injectors/README.md:106]: Ah, that's from another diff. Remember that we went away from the `asset` word and use `content` now.
I will undo this change.

Comment (vpetersson) [reply javascript-injectors/README.md:106]: We still call it assets in the API, so i think we should call it assets here too.

**Pull request merged**